extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -103,6 +107,19 @@ export class AmpSidebar extends AMP.BaseElement {
     this.registerAction('toggle', this.toggle_.bind(this));
     this.registerAction('open', this.open_.bind(this));
     this.registerAction('close', this.close_.bind(this));
+
+    installGlobalClickListenerForDoc(this.getAmpDoc()).addBeforeHandler(e => {
```

Comment:
Where is the listener removed? We don't need it working all the time, right? Only when the sidebar is open, I assume.